examples/QubitRotation.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Qubit Rotation with JAX
+# 
+# In this notebook, we shall investigate how we can use simple gradient descent to optimize for the a qubit rotation. The idea is as follows: We start in $|1 \rangle$ state on a Bloch Sphere and apply rotations around Z-Y-Z axes (in that order), each of the three with angles $\phi$, $\theta$, and $\omega$ respectively
+# 
```

Comment:
Add a simple line to describe what this R operator is. Eg.

"A general unitary operator is parameterised by three real parameters as, "